build.js

```diff
@@ -60,9 +60,11 @@ if (!fs.existsSync(binDir)) {
 
 const minidumpStackwalk = path.resolve(__dirname, 'build', 'src', 'processor', 'minidump_stackwalk') + exe
 fs.copyFileSync(minidumpStackwalk, minidumpStackwalkDest)
+fs.chmodSync(minidumpStackwalkDest, 0o755)
```

Comment:
I think only the `prepublishOnly` part of this PR is required, as when the files are built through build.js they will have the correct permissions.

Reply:
Yes, true. I will revert this

Reply:
https://github.com/electron/node-minidump/pull/43/commits/fb55d92c7d2fd382819ee729cd902285018e5cbf